src/lisk-constants/index.js

```diff
@@ -28,13 +28,6 @@ export const EPOCH_TIME = new Date(Date.UTC(2016, 4, 24, 17, 0, 0, 0));
 export const EPOCH_TIME_MILLISECONDS = EPOCH_TIME.getTime();
```

Comment:
Maybe fees are specific to the `transactions` module? They are part of the protocol though. Thoughts @Tosch110 ?

Reply:
I think it makes sense especially considering the near planned future, to put it into the `transactions` module. I can see that these constants get replaced by functions. They will be gathering data about transaction fees according to last transactions accepted by the network. Those functions will probably be part of the `transactions` module, too.